pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -251,3 +322,17 @@ func newPGOwnerReferences(pod *v1.Pod) []metav1.OwnerReference {
 	ref := metav1.NewControllerRef(pod, gvk)
 	return []metav1.OwnerReference{*ref}
 }
+
+func jobTerminated(condition batchv1.JobCondition) bool {
+	if condition.Status == v1.ConditionTrue && (condition.Type == batchv1.JobComplete || condition.Type == batchv1.JobFailed || condition.Type == batchv1.JobSuspended) {
+		return true
+	}
+	return false
+}
+
+func podTerminated(podStatus v1.PodStatus) bool {
+	if podStatus.Phase == v1.PodSucceeded || podStatus.Phase == v1.PodFailed {
```

Comment:
return `podStatus.Phase == v1.PodSucceeded || podStatus.Phase == v1.PodFailed` is ok. So does job.